docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -169,4 +445,111 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use JSON Schema file ``schema.jsonschema`` with content
+
+.. code:: sh
```

Comment:
This code block might render better as `json` instead of `sh`. (I've got a branch that I'll PR after this one merges that updates the language-highlighting hints in various places, so I'll get this one if you don't.)

Reply:
Indeed it does, thanks!